pkgs/development/mobile/androidenv/androidndk-pkgs.nix

```diff
@@ -84,6 +84,7 @@ rec {
           $out/bin/${targetPlatform.config}-g++ \
           -e '130i    extraBefore+=(-Wl,--fix-cortex-a8)' \
           -e 's|^(extraBefore=)\(\)$|\1(${builtins.toString flags})|'
+        echo "-no-pie" >> $out/nix-support/cc-cflags
```

Comment:
/cc @Ericson2314 Is this okay to do globally?

Reply:
Yeah sounds good. I'd maybe put it in the `extraBefore` thing above, but fine either way.